docs/07_tutorial4.md

```diff
@@ -155,3 +155,97 @@ So how can a machine learning model detect ink? In the electron microscope image
 Now let’s create a model! This part of the tutorial is over [on Kaggle as a notebook](https://www.kaggle.com/code/jpposma/vesuvius-challenge-ink-detection-tutorial).
 
 To run more advanced models on the [scroll segments](data_segments), check out the winning code from the [First Letters Prize](firstletters) and the [2023 Grand Prize](grandprize).
```

Comment:
Below are changes from https://github.com/ScrollPrize/scrollprize-website/pull/18. You should remove these changes from your `main` branch if you intend to push these changes separately.

Reply:
Thank you so much for these suggestions!!! and i will do this as well, i have not really ever made a pull request before so i didnt know it would sync both like this. Should have made another fork, but i'll remove from main for now.

Reply:
https://github.com/ScrollPrize/scrollprize-website/pull/18#issuecomment-1969072582
You can close this suggestion as redundant, or ignore it.

Reply:
ok i closed the gp winner addition one. so everything i think is on this one now. thanks you!

Reply:
hey daniel, i converted this to a draft as there are many more changes im making now. apologies for wasting your time here, i appreciate your help